ui-frontend/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.view.js

```diff
@@ -80,6 +81,7 @@ module.exports = Marionette.LayoutView.extend({
     this.basicTitle.show(
       new PropertyView({
         model: new Property({
+          readOnly: this.model.get('isReadOnly'),
```

Comment:
✏️ Nit-pick: for consistency's sake it might be nice to pick `isReadOnly` or `readOnly` and stick with it everywhere.